extension-style-app/src/androidTest/java/com/mapbox/maps/testapp/style/sources/generated/GeoJsonSourceTest.kt

```diff
@@ -301,14 +320,30 @@ class GeoJsonSourceTest : BaseStyleTest() {
     val testSource = geoJsonSource("testId") {
```

Comment:
```suggestion
    val testSource = geoJsonSource(SOURCE_ID) {
```
here and below

Reply:
yeah will do, although this will change all other generated files too.